src/databricks/labs/remorph/snow/snowflake.py

```diff
@@ -214,6 +214,18 @@ def _parse_tonumber(args: list) -> local_expression.ToNumber:
     return local_expression.ToNumber(this=seq_get(args, 0), expression=seq_get(args, 1))
 
 
+def _contains_expression(expression, check):
```

Comment:
Please update the argument `check` to a meaningful one.

Reply:
Done